### ISSUE TITLE
[Mod-Incompat] [1.7.4] Unable to join friends through World Host

### ISSUE THREAD

**Issue by Cixon** [Wilyicaro/Legacy-Minecraft#602]
### Legacy4J Version

[Patreon Release] 1.21-1.7.4

### Mod Loader Version

**Used NeoForge 21.1.35**

### Bug/Glitch Occurrence Context

Unable to join friends using the WorldHost mod.

Tested on the newest/patreon release for 1.21.

### Crash Report

n/a

### What happened wrong?

Title.

**Comment by Cixon** [comment]
### DEBUG (2535)
https://gist.github.com/Cixon/7bf4b7473ef33d0168bee2124e84a75f

**Comment by Wilyicaro** [comment]
Did u test without L4J?

**Comment by Wilyicaro** [comment]
In my tests, I couldn't login even without L4J, meaning it was World Host instability.

**Comment by Cixon** [comment]
> In my tests, I couldn't login even without L4J, meaning it was World Host instability.

It ran fine, I had no other mods aside from Architectury API

**Comment by Cixon** [comment]
> Did u test without L4J?

Yeah it seems with L4J my friend and I would get the ‘Connecting to Server’ thing forever until prompted with a ‘timed out’ error. Neither of us was able to connect until L4J was disabled as a active mod.

**Comment by Cixon** [comment]
I was using 1.21.1 instead of 1.21 Minecraft but I feel this is irrelevant information?

**Comment by Wilyicaro** [comment]
> > Did u test without L4J?
> 
> Yeah it seems with L4J my friend and I would get the ‘Connecting to Server’ thing forever until prompted with a ‘timed out’ error. Neither of us was able to connect until L4J was disabled as a active mod.

Elaborate further, if the host had L4J installed, you wouldn't be able to connect, right?

**Comment by Wilyicaro** [comment]
I think this is some change that World Host made in some update that created some kind of incompatibility.

**Comment by Cixon** [comment]
> Elaborate further, if the host had L4J installed, you wouldn't be able to connect, right?

In the tests conducted, both the hoster and joiner had L4J installed. This resulted in both users not being able to connect. I have not tried having one user disable while the other keeps L4J enabled.

**Comment by Wilyicaro** [comment]
maybe this is something related to https://github.com/Gaming32/world-host/issues/97 ?
idk, I didn't see anything in that log at least

**Comment by Cixon** [comment]
> maybe this is something related to [Gaming32/world-host#97](https://github.com/Gaming32/world-host/issues/97) ?
> idk, I didn't see anything in that log at least

Possibly, for now, I guess I could use e4mc but that one isn’t working for me either, could be modpack related said something about a gui..

I could force a crash.log event to see if something can get dug up for this Worldhost issue? That ticket was in July and I worry a third-party patch may need to be made.

**Comment by Cixon** [comment]
> idk, I didn't see anything in that log at least

Here's a forced crash report (holding F3+C):

[crash-2024-09-21_07.50.18-client.txt](https://github.com/user-attachments/files/17084096/crash-2024-09-21_07.50.18-client.txt)

**Comment by Wilyicaro** [comment]
Fixed in 1.7.5, thanks for reporting.
